projects/core/src/lib/constants/default-element-predicate.ts

```diff
@@ -1,5 +1,13 @@
+import {MaskitoReactContentEditableElement} from '../classes';
 import {MaskitoElementPredicate} from '../types';
 
-export const MASKITO_DEFAULT_ELEMENT_PREDICATE: MaskitoElementPredicate = e =>
-    e.querySelector<HTMLInputElement | HTMLTextAreaElement>('input,textarea') ||
-    (e as HTMLInputElement | HTMLTextAreaElement);
+export const MASKITO_DEFAULT_ELEMENT_PREDICATE: MaskitoElementPredicate = e => {
+    if (e.isContentEditable) {
+        return new MaskitoReactContentEditableElement(e).proxy;
```

Comment:
maybe it shouldn't be used here, we can try to add this logic inside mask.ts if you want it to be working everywhere